frontend/src/lib/components/metadata/MetricRange.svelte

```diff
@@ -9,24 +24,34 @@
 		</span>
 		<span
 			contenteditable="true"
-			on:blur={(e) =>
-				metricRange.update((range) => {
-					range[0] = parseFloat(e.currentTarget.innerText);
-					return [...range];
-				})}
+			role="textbox"
+			aria-label="Metric range min"
+			tabindex="0"
+			on:keydown={(e) => {
+				if (e.key === 'Enter') {
+					e.preventDefault();
+					e.currentTarget.blur();
+				}
+			}}
+			on:blur={(e) => updateRange(e, 1)}
 		>
 			{$metricRange[0].toFixed(2)}
 		</span>
 		<div class="w-10 h-4 mx-2.5 bg-gradient-to-r from-primary-light to-primary" />
-		<span
+		<div
```

Comment:
nit: why is this a div now when the other one is still a span?

Reply:
good catch